src/data/tools.json

```diff
@@ -1290,82 +1325,98 @@
   },
   {
     "id": 143,
     "title": "RapidAPI",
     "url": "https://rapidapi.com/",
     "description": "The Next-Generation API Platform.",
-    "tag": [ "API", "devtools", "web", "testing"]
+    "tag": ["API", "devtools", "web", "testing"]
   },
   {
     "id": 144,
     "title": "Streamlit",
     "url": "https://streamlit.io/",
     "description": "A faster way to build and share data apps.",
-    "tag": [ "Python", "framework", "web"]
+    "tag": ["Python", "framework", "web"]
   },
   {
     "id": 145,
     "title": "daily.dev",
     "url": "https://daily.dev/",
     "description": "An online community where developers grow together.",
     "logo": "dailydev.png",
-    "tag": [ "productivity", "tool", "community"]
+    "tag": ["productivity", "tool", "community"]
   },
   {
     "id": 146,
     "title": "Pngegg",
     "url": "https://www.pngegg.com/",
     "description": "A great place to find a wide variety of images you would desire for your designs.",
-    "tag": [ "design assets", "design", "image"]
+    "tag": ["design assets", "design", "image"]
   },
   {
     "id": 147,
     "title": "3D icons",
     "url": "https://3dicons.co/",
     "description": "Beautifully crafted open source 3D icons.",
     "logo": "3dicons.png",
-    "tag": [ "icons", "iconography", "design", "design assets" ]
+    "tag": ["icons", "iconography", "design", "design assets"]
   },
   {
     "id": 148,
     "title": "snippet generator",
     "url": "https://snippet-generator.app/",
     "description": "Snippet generator for Visual Studio Code, Sublime Text and Atom.",
-    "tag": [ "snippet", "tools", "utility", "generator" ]
+    "tag": ["snippet", "tools", "utility", "generator"]
   },
   {
     "id": 149,
     "title": "clientdb",
     "url": "https://clientdb.dev/",
     "description": "Build lightning fast apps with instantaneous interactions.",
     "logo": "clientdb.png",
-    "tag": [ "database", "tool"]
+    "tag": ["database", "tool"]
   },
   {
     "id": 150,
     "title": "snoopForms",
     "url": "https://snoopforms.com/",
     "description": "The Open-source Typeform Alternative.",
     "logo": "snoop-forms.png",
-    "tag": [ "no-code", "tool", "analytics", "data platform", "open source"]
+    "tag": ["no-code", "tool", "analytics", "data platform", "open source"]
   },
   {
     "id": 151,
     "title": "Medusa",
     "url": "https://medusajs.com/",
     "description": "The open-source Shopify alternative.",
     "logo": "medusa.png",
-    "tag": [ "development platform", "tool", "API", "open source"]
+    "tag": ["development platform", "tool", "API", "open source"]
+  },
+  {
+    "id": 152,
+    "title": "CSS Gradient Generator",
+    "url": "https://www.joshwcomeau.com/gradient-generator/",
+    "description": "Beautiful, lush gradients by Josh W. Comeau.",
+    "tag": [
+      "css",
+      "tool",
+      "tools",
+      "design",
+      "generator",
+      "generators",
+      "utility",
+      "utilities",
+      "online tools"
```

Comment:
I wonder whether we should add gradient generator?

Reply:
I think that would be a good idea because it would work if the tool chest grows :)

Reply:
Added the tag!